package.json

```diff
@@ -3,9 +3,9 @@
   "version": "0.2.0",
   "description": "responsive <img> tag with cleaner srcset/sizes interface.",
   "main": "lib/Image.js",
+  "types": "lib/Image.d.ts",
```

Comment:
it is recommended that `types` field should be added at the official docs:

https://github.com/Microsoft/TypeScript-Handbook/blob/master/pages/declaration%20files/Publishing.md#including-declarations-in-your-npm-package